### ISSUE TITLE
a11y: Icon widget accessibility

### ISSUE THREAD

**Issue by ramiy** [pojome/elementor#3268]
## Description

The icon widget need some accessibility love!

The current `<i>` tag should have an aria tags for better accessibility.  As icons are used for pure decoration or visual styling they should not be displayed to an assistive technology-using user. We need to make sure it's not read, by adding the `aria-hidden="true"` to the markup.

![icon-widget](https://user-images.githubusercontent.com/576623/35198743-16531cea-fefc-11e7-81d9-5a58beb19895.png)
